tests/dags/common/test_dag_factory.py

```diff
@@ -47,11 +47,18 @@ def test_push_output_paths_wrapper(func, media_types, stores):
         ti_mock,
         args=[func_mock, value],
     )
-    assert ti_mock.xcom_push.call_count == len(
-        media_types
-    ), "# of output paths didn't match # of stores expected"
-    for args, store in zip(ti_mock.xcom_push.calls, stores):
```

Comment:
This was a fun bug, so for anyone interested: this had to be updated from `xcom_push.calls` -> `xcom_push.mock_calls`, or else _calls were actually added_ in the slicing/assertion. Ie after exiting the loop, `mock_calls` was:

```
[call(key='image_tsv', value='/var/workflow_output/None_image_v001_20220218234009.tsv'),
call(key='duration', value=0.00024759999359957874),
call.calls.__getitem__(slice(None, -1, None)),
call.calls.__getitem__().__iter__()]
```
😄

Reply:
Oh jeez, that must have been an awful one to find! :cold_sweat: